<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>USFM Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_Text_1">
      <property name="text">
       <string>&lt;b&gt;Text 1&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_Text_2">
      <property name="text">
       <string>&lt;b&gt;Text 2&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTextEdit" name="textEdit_Text_1">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTextEdit" name="textEdit_Text_2">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QTextEdit" name="textEdit_Text_3">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_Text_3">
      <property name="text">
       <string>&lt;b&gt;Text 3&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_dir_1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_dir_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_dir_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionNew_Text_1"/>
     <addaction name="actionNew_Text_2"/>
     <addaction name="actionNew_Text_3"/>
    </widget>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="actionOpen_Text_1"/>
     <addaction name="actionOpen_Text_2"/>
     <addaction name="actionOpen_Text_3"/>
    </widget>
    <widget class="QMenu" name="menuSave">
     <property name="title">
      <string>Save</string>
     </property>
     <addaction name="actionSave_Text_1"/>
     <addaction name="actionSave_Text_2"/>
     <addaction name="actionSave_Text_3"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="separator"/>
    <addaction name="menuOpen"/>
    <addaction name="menuSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuUndo">
     <property name="title">
      <string>Undo</string>
     </property>
     <addaction name="actionUndo_in_Text_1"/>
     <addaction name="actionUndo_in_Text_2"/>
     <addaction name="actionUndo_in_Text_3"/>
    </widget>
    <widget class="QMenu" name="menuRedo">
     <property name="title">
      <string>Redo</string>
     </property>
     <addaction name="actionRedo_in_Text_1"/>
     <addaction name="actionRedo_in_Text_2"/>
     <addaction name="actionRedo_in_Text_3"/>
    </widget>
    <widget class="QMenu" name="menuCut">
     <property name="title">
      <string>Cut</string>
     </property>
     <addaction name="actionCut_from_Text_1"/>
     <addaction name="actionCut_from_Text_2"/>
     <addaction name="actionCut_from_Text_3"/>
    </widget>
    <widget class="QMenu" name="menuCopy">
     <property name="title">
      <string>Copy</string>
     </property>
     <addaction name="actionCopy_from_Text_1"/>
     <addaction name="actionCopy_from_Text_2"/>
     <addaction name="actionCopy_from_Text_3"/>
    </widget>
    <widget class="QMenu" name="menuPaste">
     <property name="title">
      <string>Paste</string>
     </property>
     <addaction name="actionPaste_into_Text_1"/>
     <addaction name="actionPaste_into_Text_2"/>
     <addaction name="actionPaste_into_Text_3"/>
    </widget>
    <widget class="QMenu" name="menuSelect_All">
     <property name="title">
      <string>Select All</string>
     </property>
     <addaction name="actionSelect_All_in_Text_1"/>
     <addaction name="actionSelect_All_in_Text_2"/>
     <addaction name="actionSelect_All_in_Text_3"/>
    </widget>
    <widget class="QMenu" name="menuFind">
     <property name="title">
      <string>Find</string>
     </property>
     <addaction name="actionFind_in_Text_1"/>
     <addaction name="actionFind_in_Text_2"/>
     <addaction name="actionFind_in_Text_3"/>
    </widget>
    <widget class="QMenu" name="menuFind_and_Replace">
     <property name="title">
      <string>Find and Replace</string>
     </property>
     <addaction name="actionFind_and_Replace_in_Text_1"/>
     <addaction name="actionFind_and_Replace_in_Text_2"/>
     <addaction name="actionFind_and_Replace_in_Text_3"/>
    </widget>
    <widget class="QMenu" name="menuFind_and_Replace_All">
     <property name="title">
      <string>Find and Replace All</string>
     </property>
     <addaction name="actionFind_and_Replace_All_in_Text_1"/>
     <addaction name="actionFind_and_Replace_All_in_Text_2"/>
     <addaction name="actionFind_and_Replace_All_in_Text_3"/>
    </widget>
    <addaction name="menuUndo"/>
    <addaction name="menuRedo"/>
    <addaction name="separator"/>
    <addaction name="menuCut"/>
    <addaction name="menuCopy"/>
    <addaction name="menuPaste"/>
    <addaction name="menuSelect_All"/>
    <addaction name="separator"/>
    <addaction name="menuFind"/>
    <addaction name="menuFind_and_Replace"/>
    <addaction name="menuFind_and_Replace_All"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuFont">
     <property name="title">
      <string>Font</string>
     </property>
     <addaction name="actionFont_of_Text_1"/>
     <addaction name="actionFont_of_Text_2"/>
     <addaction name="actionFont_of_Text_3"/>
    </widget>
    <widget class="QMenu" name="menuColor">
     <property name="title">
      <string>Color</string>
     </property>
     <addaction name="actionColor_of_Text_1"/>
     <addaction name="actionColor_of_Text_2"/>
     <addaction name="actionColor_of_Text_3"/>
    </widget>
    <widget class="QMenu" name="menuHighlight">
     <property name="title">
      <string>Highlight</string>
     </property>
     <addaction name="actionHighlight_in_Text_1"/>
     <addaction name="actionHighlight_in_Text_2"/>
     <addaction name="actionHighlight_in_Text_3"/>
    </widget>
    <widget class="QMenu" name="menuInterface_Language">
     <property name="title">
      <string>Interface Language</string>
     </property>
     <addaction name="actionEnglish"/>
     <addaction name="actionRussian"/>
    </widget>
    <addaction name="menuFont"/>
    <addaction name="menuColor"/>
    <addaction name="menuHighlight"/>
    <addaction name="separator"/>
    <addaction name="menuInterface_Language"/>
    <addaction name="separator"/>
    <addaction name="actionShow_file_paths_and_names"/>
    <addaction name="actionHide_file_paths_and_names"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuZoom_In">
     <property name="title">
      <string>Zoom In</string>
     </property>
     <addaction name="actionZoom_In_Text_1"/>
     <addaction name="actionZoom_In_Text_2"/>
     <addaction name="actionZoom_In_Text_3"/>
    </widget>
    <widget class="QMenu" name="menuZoom_Out">
     <property name="title">
      <string>Zoom Out</string>
     </property>
     <addaction name="actionZoom_Out_Text_1"/>
     <addaction name="actionZoom_Out_Text_2"/>
     <addaction name="actionZoom_Out_Text_3"/>
    </widget>
    <widget class="QMenu" name="menuFormatted_USFM">
     <property name="title">
      <string>Formatted USFM</string>
     </property>
     <addaction name="actionFormatted_Text_1"/>
     <addaction name="actionFormatted_Text_2"/>
     <addaction name="actionFormatted_Text_3"/>
    </widget>
    <widget class="QMenu" name="menuUnformatted_USFM">
     <property name="title">
      <string>Unformatted USFM</string>
     </property>
     <addaction name="actionUnformatted_Text_1"/>
     <addaction name="actionUnformatted_Text_2"/>
     <addaction name="actionUnformatted_Text_3"/>
    </widget>
    <addaction name="menuZoom_In"/>
    <addaction name="menuZoom_Out"/>
    <addaction name="separator"/>
    <addaction name="menuFormatted_USFM"/>
    <addaction name="menuUnformatted_USFM"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew_Text_1">
   <property name="text">
    <string>New Text 1</string>
   </property>
  </action>
  <action name="actionNew_Text_2">
   <property name="text">
    <string>New Text 2</string>
   </property>
  </action>
  <action name="actionOpen_Text_1">
   <property name="text">
    <string>Open Text 1</string>
   </property>
  </action>
  <action name="actionOpen_Text_2">
   <property name="text">
    <string>Open Text 2</string>
   </property>
  </action>
  <action name="actionSave_Text_1">
   <property name="text">
    <string>Save Text 1</string>
   </property>
  </action>
  <action name="actionSave_Text_2">
   <property name="text">
    <string>Save Text 2</string>
   </property>
  </action>
  <action name="actionUndo_in_Text_1">
   <property name="text">
    <string>Undo in Text 1</string>
   </property>
  </action>
  <action name="actionUndo_in_Text_2">
   <property name="text">
    <string>Undo in Text 2</string>
   </property>
  </action>
  <action name="actionRedo_in_Text_1">
   <property name="text">
    <string>Redo in Text 1</string>
   </property>
  </action>
  <action name="actionRedo_in_Text_2">
   <property name="text">
    <string>Redo in Text 2</string>
   </property>
  </action>
  <action name="actionCut_from_Text_1">
   <property name="text">
    <string>Cut from Text 1</string>
   </property>
  </action>
  <action name="actionCut_from_Text_2">
   <property name="text">
    <string>Cut from Text 2</string>
   </property>
  </action>
  <action name="actionCopy_from_Text_1">
   <property name="text">
    <string>Copy from Text 1</string>
   </property>
  </action>
  <action name="actionCopy_from_Text_2">
   <property name="text">
    <string>Copy from Text 2</string>
   </property>
  </action>
  <action name="actionPaste_into_Text_1">
   <property name="text">
    <string>Paste into Text 1</string>
   </property>
  </action>
  <action name="actionPaste_into_Text_2">
   <property name="text">
    <string>Paste into Text 2</string>
   </property>
  </action>
  <action name="actionFont_of_Text_1">
   <property name="text">
    <string>Font of Text 1</string>
   </property>
  </action>
  <action name="actionFont_of_Text_2">
   <property name="text">
    <string>Font of Text 2</string>
   </property>
  </action>
  <action name="actionColor_of_Text_1">
   <property name="text">
    <string>Color of Text 1</string>
   </property>
  </action>
  <action name="actionColor_of_Text_2">
   <property name="text">
    <string>Color of Text 2</string>
   </property>
  </action>
  <action name="actionHighlight_in_Text_1">
   <property name="text">
    <string>Highlight in Text 1</string>
   </property>
  </action>
  <action name="actionHighlight_in_Text_2">
   <property name="text">
    <string>Highlight in Text 2</string>
   </property>
  </action>
  <action name="actionZoom_In_Text_1">
   <property name="text">
    <string>Zoom In Text 1</string>
   </property>
  </action>
  <action name="actionZoom_In_Text_2">
   <property name="text">
    <string>Zoom In Text 2</string>
   </property>
  </action>
  <action name="actionZoom_Out_Text_1">
   <property name="text">
    <string>Zoom Out Text 1</string>
   </property>
  </action>
  <action name="actionZoom_Out_Text_2">
   <property name="text">
    <string>Zoom Out Text 2</string>
   </property>
  </action>
  <action name="actionSelect_All_in_Text_1">
   <property name="text">
    <string>Select All in Text 1</string>
   </property>
  </action>
  <action name="actionSelect_All_in_Text_2">
   <property name="text">
    <string>Select All in Text 2</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNew_Text_3">
   <property name="text">
    <string>New Text 3</string>
   </property>
  </action>
  <action name="actionOpen_Text_3">
   <property name="text">
    <string>Open Text 3</string>
   </property>
  </action>
  <action name="actionSave_Text_3">
   <property name="text">
    <string>Save Text 3</string>
   </property>
  </action>
  <action name="actionUndo_in_Text_3">
   <property name="text">
    <string>Undo in Text 3</string>
   </property>
  </action>
  <action name="actionRedo_in_Text_3">
   <property name="text">
    <string>Redo in Text 3</string>
   </property>
  </action>
  <action name="actionCut_from_Text_3">
   <property name="text">
    <string>Cut from Text 3</string>
   </property>
  </action>
  <action name="actionCopy_from_Text_3">
   <property name="text">
    <string>Copy from Text 3</string>
   </property>
  </action>
  <action name="actionPaste_into_Text_3">
   <property name="text">
    <string>Paste into Text 3</string>
   </property>
  </action>
  <action name="actionSelect_All_in_Text_3">
   <property name="text">
    <string>Select All in Text 3</string>
   </property>
  </action>
  <action name="actionZoom_In_Text_3">
   <property name="text">
    <string>Zoom In Text 3</string>
   </property>
  </action>
  <action name="actionZoom_Out_Text_3">
   <property name="text">
    <string>Zoom Out Text 3</string>
   </property>
  </action>
  <action name="actionFont_of_Text_3">
   <property name="text">
    <string>Font of Text 3</string>
   </property>
  </action>
  <action name="actionColor_of_Text_3">
   <property name="text">
    <string>Color of Text 3</string>
   </property>
  </action>
  <action name="actionHighlight_in_Text_3">
   <property name="text">
    <string>Highlight in Text 3</string>
   </property>
  </action>
  <action name="actionFormatted_Text_1">
   <property name="text">
    <string>Formatted Text 1</string>
   </property>
  </action>
  <action name="actionFormatted_Text_2">
   <property name="text">
    <string>Formatted Text 2</string>
   </property>
  </action>
  <action name="actionFormatted_Text_3">
   <property name="text">
    <string>Formatted Text 3</string>
   </property>
  </action>
  <action name="actionUnformatted_Text_1">
   <property name="text">
    <string>Unformatted Text 1</string>
   </property>
  </action>
  <action name="actionUnformatted_Text_2">
   <property name="text">
    <string>Unformatted Text 2</string>
   </property>
  </action>
  <action name="actionUnformatted_Text_3">
   <property name="text">
    <string>Unformatted Text 3</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="actionFind_in_Text_1">
   <property name="text">
    <string>Find in Text 1</string>
   </property>
  </action>
  <action name="actionFind_in_Text_2">
   <property name="text">
    <string>Find in Text 2</string>
   </property>
  </action>
  <action name="actionFind_in_Text_3">
   <property name="text">
    <string>Find in Text 3</string>
   </property>
  </action>
  <action name="actionFind_and_Replace_in_Text_1">
   <property name="text">
    <string>Find and Replace in Text 1</string>
   </property>
  </action>
  <action name="actionFind_and_Replace_in_Text_2">
   <property name="text">
    <string>Find and Replace in Text 2</string>
   </property>
  </action>
  <action name="actionFind_and_Replace_in_Text_3">
   <property name="text">
    <string>Find and Replace in Text 3</string>
   </property>
  </action>
  <action name="actionFind_and_Replace_All_in_Text_1">
   <property name="text">
    <string>Find and Replace All in Text 1</string>
   </property>
  </action>
  <action name="actionFind_and_Replace_All_in_Text_2">
   <property name="text">
    <string>Find and Replace All in Text 2</string>
   </property>
  </action>
  <action name="actionFind_and_Replace_All_in_Text_3">
   <property name="text">
    <string>Find and Replace All in Text 3</string>
   </property>
  </action>
  <action name="actionShow_file_paths_and_names">
   <property name="text">
    <string>Show file paths and names</string>
   </property>
  </action>
  <action name="actionHide_file_paths_and_names">
   <property name="text">
    <string>Hide file paths and names</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
